content/en/docs/ambient/architecture/data-plane/index.md

```diff
@@ -46,8 +46,26 @@ requests directly to the destination without going through any waypoint proxy ev
 Currently, traffic from sidecars and gateways won't go through any waypoint proxy either and they will be made aware of waypoint proxies
 in a future release.
 
-#### Dataplane details
+#### L4 dataplane details
 
+##### Identity
+
+All inbound and outbound traffic between workloads in the ambient mesh is secured by the data plane, using mTLS via {{< gloss >}}HBONE{{< /gloss >}}, ztunnel, and x509 certificates.
+
+However, the certificates used will be bound to the workloads themselves, not ztunnel. This means ztunnel will hold multiple distinct certificates at a time, one for each unique identity (service account) running on its node.
```

Comment:
This is a tough bit of information to convey. I'm not certain the suggestion should be accepted as is but perhaps it will contain inspiration for a revision at least.

```suggestion
All inbound and outbound traffic between workloads in the ambient mesh is secured by the data plane, using mTLS via {{< gloss >}}HBONE{{< /gloss >}}, ztunnel, and x509 certificates. The x509 certificates will be used by to sign traffic on behalf of node-local pods as well as validate traffic arriving at these pods. In order to achieve a zero-trust posture the certificates used will not bear the identity of a trusted node-proxy component instead containing the attested identity of each node-local pod respectively. This requires the ztunnel to manage multiple distinct certificates at a time, one for each unique identity (service account) for every node-local pod.
```

Reply:
Yeah I'm kinda trying to avoid getting too much into "how mTLS works" here since we cover that more in the HBONE arch bit (which we breadcrumb to via the glossary ref).

Reply:
Reworded a bit using some of your suggestion.

Reply:
Reworded a bit using some of your suggested text.

Reply:
nice, good middle ground